packages/langium/src/serializer/json-serializer.ts

```diff
@@ -30,6 +32,13 @@ export interface AstNodeWithTextRegion extends AstNode {
     $textRegion?: AstNodeRegionWithAssignments;
 }
 
+/**
+ * {@link AstNode}s that may carry a comment CST node in front of itself.
+ */
```

Comment:
Suggestion: Given that the `CommentProvider` might return any kind of different comment (only the default is "right in front of the node"):

```suggestion
/**
 * {@link AstNode}s that may carry a semantically relevant comment.
 */
```